hack/update-openapi-spec.sh

```
@@ -0,0 +1,340 @@
+  REGISTRY_AUTH_OPTS="--registry-config=${REGISTRY_AUTH_FILE}"
+else
+  echo "WARNING: No pull secret found. Proceeding without authentication."
+  echo "This may fail if accessing private registries."
+fi
+
+echo "Extracting etcd from OpenShift release ${OPENSHIFT_RELEASE}..."
+ETCD="${BIN_DIR}/etcd"
+ETCD_EXTRACT_DIR="${TMP_DIR}/etcd-extract"
+rm -rf "${ETCD_EXTRACT_DIR}"
+mkdir -p "${ETCD_EXTRACT_DIR}"
+ETCD_IMAGE=$(oc adm release info "${OPENSHIFT_RELEASE}" ${REGISTRY_AUTH_OPTS} --image-for=etcd)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)